pkg/webui/console/views/application-data/index.js

```diff
@@ -0,0 +1,61 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { defineMessages } from 'react-intl'
+import { Container, Col, Row } from 'react-grid-system'
+
+import sharedMessages from '../../../lib/shared-messages'
+import Breadcrumb from '../../../components/breadcrumbs/breadcrumb'
+import { withBreadcrumb } from '../../../components/breadcrumbs/context'
+import Message from '../../../lib/components/message'
+import ApplicationEvents from '../../containers/application-events'
+
+const m = defineMessages({
+  appData: 'Application data',
+})
+
+@withBreadcrumb('apps.single.data', function (props) {
+  return (
+    <Breadcrumb
+      path={`/console/applications/${props.appId}/data`}
+      icon="data"
+      content={sharedMessages.data}
+    />
+  )
+})
+export default class Data extends React.Component {
+
+  render () {
+    const { appId } = this.props.match.params
+
+    return (
```

Comment:
Please add a `<ReactHelmet />` entry for the view. In general, a lot of application pages still lack these. Can you follow up with an issue?

Reply:
Never mind, I'll create an issue for this.